src/lib/openapi/spec/tag-type-schema.ts

```diff
@@ -4,17 +4,24 @@ export const tagTypeSchema = {
     $id: '#/components/schemas/tagTypeSchema',
     type: 'object',
     additionalProperties: false,
+    description: 'A tag type.',
     required: ['name'],
     properties: {
         name: {
             type: 'string',
+            description: 'The name of the tag type.',
+            example: 'color',
         },
         description: {
             type: 'string',
+            description: 'The description of the tag type.',
+            example: 'A tag type for describing the color of a tag.',
         },
         icon: {
             type: 'string',
             nullable: true,
+            description: 'The icon of the tag type.',
+            example: 'not-really-used',
```

Comment:
This example is a little confusing. Are you saying we don't use the icons provided? If they provide icons, what format should it be in?

Reply:
We're definitely not using them in the admin UI 

https://sandbox.getunleash.io/enterprise/tag-types/create
![image](https://github.com/Unleash/unleash/assets/455064/1dcb4302-acee-4827-a782-931516b18323)

https://sandbox.getunleash.io/enterprise/tag-types/edit/group
![image](https://github.com/Unleash/unleash/assets/455064/5f00f4c3-39e1-4299-8a1f-a3cf27164cc3)

But icons do exist in our database and in our API. So I think this could be just an idea that never got implemented. 

Maybe @chriswk who worked on this 2 years ago might remember something.

Reply:
What Gaston said, it was meant to be an extra feature so you could paste an URL to a valid icon, so that :slack: / :jira:  had a more obvious illustration than just the first letter of the type capitalised.

Reply:
I don't think it ever got prioritised.

Reply:
Got it. Do we have a format for that in any way?

Based on that, I'd definitely put that in the description and maybe even deprecate the properties?

Reply:
Right now it can be anything, it will depend on what we want to do when we finally implement this. It's like predicting the future... we can tell customers to write it as a full-url, but maybe in the future we want to use an enum that is not a URL. So :shrug: 

Deprecating it also doesn't feel right because it's like some feature that has always been a work in progress. Maybe we can hide it from the documentation, but it was already there... so this is one of those unfortunate cases where we've dropped the ball and we didn't know about it. 

We can update the description, but also the example helps documenting that it's not being used

Reply:
Yeah, that sounds sensible. I'd add a note to the description that this isn't used yet, but might be in the future, then.